### ISSUE TITLE
Job stuck in "RUNNING" status

### ISSUE THREAD

**Issue by majyphi** [rkanter/MiniOozieExample#2]
Hi,

Not sure if this is still maintained but I tried running the example "as is", without modification (except the workaround stated in the README.md), and MiniOozie starts with no problem, but the job does not seems to actually run.

The only info I get is the status : "RUNNING"
But it stays like that for hours, even with a smaller sample text in data.txt.

Any idea what I got wrong ?

**Comment by rkanter** [comment]
Hi,

I hadn't run this in quite a while, but I just tried it now with the default versions of Oozie and Hadoop is it works fine.  What versions are you using?  Do the default versions work for you?

Do you see any errors or other issues in the output before the "RUNNING" part?  For example, if Hadoop is in some funny state, it could be that the underlying Hadoop job is simply waiting to start for some reason.

Another thing you could look at is the Oozie and YARN Web UIs, which may provide more details on what's happening.  It should be possible to get to them if you find the port numbers in the output.  Try searching for "Jetty bound to port" or simply "http" to find the port numbers.

**Comment by majyphi** [comment]
Hi,

I digged a bit, checked the different ports, and it seemed to work fine on Hadoop and Yarn side. However impossible to get Oozie Web UI. Jetty would throw a 404.
And indeed, just before submitting the job I got these lines :

> 16:39:05,396  INFO Services:520 - Initialized
> 16:39:05,397  INFO Services:520 - Running with JARs for Hadoop version [2.6.0-cdh5.10.0]
> 16:39:05,397  INFO Services:520 - Oozie System ID [oozie-gfa] started!
> 16:39:05,504  INFO Services:520 - Shutdown

So it seems to shutdown just after setup.

However, by looking for a solution, I resorted to using directly the [WorkflowTest.java](https://github.com/apache/oozie/blob/branch-4.1/minitest/src/test/java/org/apache/oozie/test/WorkflowTest.java), which also extends MiniOozieTestCase... And it works for me. ¯\\\_(ツ)\_/¯
Not sure where the difference lies, though.

I will just work from there (which is testing a custom ActionExecutor).

Thanks for your help! And sorry for the trouble.
